libs/liburing/Makefile

```diff
@@ -0,0 +1,65 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=liburing
+PKG_VERSION:=0.6
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://git.kernel.dk/cgit/liburing/snapshot
+PKG_HASH:=44c99b9f148a885d882acd7aa63658141675eae251298cbf79bb9a4ab307ef9c
+
+PKG_MAINTAINER:=Christian Lachner <gladiac@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/liburing
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=liburing
+  URL:=https://git.kernel.dk/cgit/liburing
+endef
+
+define Package/liburing/description
+  liburing provides helpers to setup and teardown io_uring instances,
+  and also a simplified interface for applications that don't need
+  (or want) to deal with the full kernel side implementation.
+  For more info on io_uring, please see: https://kernel.dk/io_uring.pdf
+endef
+
+CONFIGURE_ARGS:=--prefix=$(CONFIGURE_PREFIX) --cc=${TARGET_CC}
+
+ifeq ($(CONFIG_USE_MUSL),y)
+	TARGET_CFLAGS+=-Dloff_t=off_t
+endif
+
+ifeq ($(CONFIG_USE_UCLIBC),y)
+	TARGET_CFLAGS+=-fno-stack-protector
```

Comment:
Is this really necessary?

https://github.com/openwrt/openwrt/commit/b933f9cf0cb254e368027cad6d5799e45b237df5 should have fixed this.

Wipe and rebuild your toolchain to make sure.

Reply:
I pulled from the master branch and also updated the feeds before I started working on this. However, I will check if -fno-stack-protecter is still necessary today. I will start from zero to make sure. Maybe reusing old .config-files is causing issues here...

Reply:
@neheb Ok, I just checked and you were right. I will reenable the stack-protector for uclibc targets.